### PR TITLE
Release 100.14.1.1

### PR DESCRIPTION
**Pull request by vquach2404**
## Description

This PR merges `v.next` into `main` for 100.14.1.1 release

## Linked Issue(s)

- `common-samples/issues/3524`

---

Release notes since 100.14.0, #1164 

#### New samples

- [**Set max extent**](https://developers.arcgis.com/ios/swift/sample-code/set-max-extent)
- [**Display route layer**](https://developers.arcgis.com/ios/swift/sample-code/display-route-layer)
- [**Apply unique values with alternate symbols**](https://developers.arcgis.com/ios/swift/sample-code/apply-unique-values-with-alternate-symbols)

#### Enhancements

- Update [**Display feature layers**](https://developers.arcgis.com/ios/swift/sample-code/display-feature-layers)
to use new feature layer constructor.
- Add new "Favorites" category to access favorited samples quickly.

#### Bug fixes

- Fixed popup with animation statistics for [**Animate 3D graphic**](https://developers.arcgis.com/ios/swift/sample-code/animate-3d-graphic).
- Leveled initial scene viewpoint in [**Explore scenes in flyover AR**](https://developers.arcgis.com/ios/swift/sample-code/explore-scenes-in-flyover-ar).
- Fixed authentication prompt for [**Token authentication**](https://developers.arcgis.com/ios/swift/sample-code/token-authentication).

### CODE REVIEW

**Review by saratchandrakarumuri — DISMISSED** [review]
Since all the changes are already verified while merging into `v.next` approving this PR

File: podfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-platform :ios, '13.0'
+platform :ios, '14.0'
 target 'ArcGIS Runtime SDK Samples' do
   use_frameworks!
   pod 'ArcGIS-Runtime-Toolkit-iOS', '~> 100.14'
```
Comment (yo1995) [inline podfile:4]: Should we bump this up to `100.14.1`? https://cocoapods.org/pods/ArcGIS-Runtime-SDK-iOS

Comment (vquach2404) [reply podfile:4]: Ahh yes, forgot about that

Comment (vquach2404) [reply podfile:4]: I tried but it said "could not apply suggestion" 😢

**Review by yo1995 — APPROVED** [review]

**Review by saratchandrakarumuri — APPROVED** [review]

**Pull request merged**